### PR TITLE
Feature 8: User can delete their current projects (Nowwws)

### PR DESCRIPTION
**Pull request by AnthonyVNgo**
Kapture 2022-06-26 at 20 51 00

### CODE REVIEW

File: client/pages/update-my-now.jsx [hunk 1]
```diff
@@ -94,7 +95,21 @@ export default class UpdateMyNow extends React.Component {
...
+    fetch(`api/now-entry/${EntryId}`, init)
+      .then(res => {})
+      .then(finalResponse => {});
+  }
+
   render() {
+    // console.log(this.context.route)
     if (!this.context.user) return <Redirect to="sign-in" />;
     if (!this.state.user) return null;
     if (!this.state.user) return <Redirect to="sign-in" />;
```
Comment (thebearingedge) [inline client/pages/update-my-now.jsx:115]: You can remove one of these user guards. Only the first one is happening

File: client/pages/update-my-now.jsx [hunk 2]
```diff
@@ -94,7 +95,21 @@ export default class UpdateMyNow extends React.Component {
...
+      method: 'DELETE',
+      headers: {
+        'X-Access-Token': window.localStorage.getItem('react-context-jwt')
+      }
+    };
+
+    fetch(`api/now-entry/${EntryId}`, init)
+      .then(res => {})
+      .then(finalResponse => {});
+  }
```
Comment (thebearingedge) [inline client/pages/update-my-now.jsx:109]: in your `.then()` on line 107 you need to filter your nowEntry array to exclude the deleted one and update the state to re-render the entries.

Comment (thebearingedge) [reply client/pages/update-my-now.jsx:109]: Actually, I think the main issue is that your NowEntryList component isn't pulling its weight. Just map the Now Entry list items directly in this component's render method.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Ok, please see my comments and also scan your pull request for commented lines that should be deleted.

https://github.com/AnthonyVNgo/final-project/pull/22/files

**Comment by AnthonyVNgo** [comment]
Kapture 2022-06-28 at 02 19 40

**Comment by AnthonyVNgo** [comment]
Kapture 2022-06-28 at 10 57 36

**Pull request merged**